frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -79,6 +79,15 @@ export interface DashCardProps {
   navigateToNewCardFromDashboard?: (
     opts: NavigateToNewCardFromDashboardOpts,
   ) => void;
+  fetchCardData: (
```

Comment:
This prop is never used, we should not be passing it in `DashboardGrid` or unit tests.